compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -2064,4 +2064,11 @@ object messages {
     val explanation =
       hl"An object that contains ${"@static"} members must have a companion class."
   }
+
+  case class SymbolIsNotAValue(symbol: Symbol)(implicit ctx: Context) extends Message(SymbolIsNotAValueID) {
+    val msg = hl"${symbol.show} is not a value"
+    val kind = "Type Mismatch"
+    val explanation =
+      hl"Scala or Java packages cannot be assigned to a value"
```

Comment:
Scala packages can be used as values if they have a package object.

`Java statics and packages cannot be used value.`

Reply:
What should be the explanation then?
What are Java statics in this context? I guess not static fields because """object O {val a = Math.PI}""" compiles.

Reply:
I found the relevant  [conversation](https://github.com/lampepfl/dotty/pull/1787). I will write a better explanation based on this.

Reply:
I updated the explanation.